### DISCUSSION TITLE
integrated intensities fo a fitted NMR spectrum

### DISCUSSION THREAD

**Discussion by ckouvatas** [spectrochempy/spectrochempy#707] [Q&A]
Dear all,
I performed a fit of NMR spectra (with signal overlapping, so not possible to just integrate a region).
The returned values are amplitudes, not integrated intensities, right?
Is there a block in the  in the fitting script (or an option in the "optimize" obect)  that we can add in order to access the integrated intensities?
Or in the 
Thank you!
Cassandre

**Comment by fernandezc** [comment]
you can change the "amplitude_mode" parameter:

see https://www.spectrochempy.fr/latest/reference/generated/spectrochempy.Optimize.html#spectrochempy.Optimize

**Reply by fernandezc** [reply]
amplitude_mode (any value of [ 'area' , 'height' ], optional, default: 'height') – Initial amplitude setting mode.
